pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/KitRequestShipping.java

```diff
@@ -1357,8 +1357,44 @@ public static Address getToAddressId(@NonNull EasyPostUtil easyPostUtil, KitRequ
             if (StringUtils.isNotBlank(addressId)) {
                 toAddress = easyPostUtil.getAddress(addressId);
             } else if (participant != null) {
+                boolean hasCareOf = kitRequestSettings.getHasCareOF() == 1;
```

Comment:
if hasCareOf is not used anywhere else, you could put that `kitRequestSettings.getHasCareOF()` directly in the if